bin/check-single-dataset.js

```diff
@@ -27,6 +27,16 @@ const checkForError = (fileName, json, schemaFileName) => {
     if (result.featureKeysError) {
       datasetsError = result.featureKeysError;
       errorMsg = result.keysErrorMsg;
+    };
+    for (let featureDef of featureDefsData) {
+      if (featureDef.options){
+        const optionsArray = Object.values(featureDef.options);
+        const results = utils.validateKeyInOptions(optionsArray);
+        if (results.optionKeyError) {
+          datasetsError = results.optionKeyError;
+          errorMsg = results.optionKeyErrorMsg;
```

Comment:
Nit: Maybe make how you access the return value (destructuring vs. using `results`) consistent between the two usages?

Reply:
Good point! Just fixed, thank you!